include/posix/pthread.h

```diff
@@ -54,8 +54,8 @@ struct posix_thread {
 
 /* Pthread cancellation */
 #define _PTHREAD_CANCEL_POS	0
-#define PTHREAD_CANCEL_ENABLE	(0 << _PTHREAD_CANCEL_POS)
-#define PTHREAD_CANCEL_DISABLE	(1 << _PTHREAD_CANCEL_POS)
+#define PTHREAD_CANCEL_ENABLE	BIT(_PTHREAD_CANCEL_POS)
+#define PTHREAD_CANCEL_DISABLE	BIT(_PTHREAD_CANCEL_POS)
```

Comment:
This is changing the value of `PTHREAD_CANCEL_ENABLE`

Reply:
@MaureenHelm indeed, thanks for pointing it out.